src/despacer.c

```diff
@@ -0,0 +1,1237 @@
+#ifndef DESPACER_H
+#define DESPACER_H
+
+#include <stdint.h>
+#include <stddef.h>
+
+#include "./despacer_tables.h"
```

Comment:
This is the only line I changed in this file. I used `./` to make it explicit.

Reply:
Now I have removed `static` and `inline` from the functions, so they are exported from the static library.

Reply:
I added the assert header too, which was being used without including it.

Reply:
I added string.h to support building on MacOS.